ssl_website_check.sh

```diff
@@ -44,11 +55,19 @@ function check_ssl_cert()
     printf "| %30s | %5s | %-13s | %-40s | %-50s |\n" "$host" "$port" "$end_date" "$serial" "${issuer:0:50}"
 }
 
+[ ! -e /usr/bin/openssl ] && { echo "The package openssl is not installed!"; exit; }
```

Comment:
Nice. Ideally we should add exit code `1` to signal the caller something went wrong.